code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -508,12 +508,19 @@
 		R.model.basic_modules += SP
 		R.model.add_module(SP, FALSE, TRUE)
 
+		for(var/obj/item/surgical_drapes/drapes in R.model.modules)
+			R.model.remove_module(drapes, TRUE)
```

Comment:
ehhh i'd rather not remove the drapes in case new people still want to do simple stuff like tend brute/tend burns without having the meta-info that surgical processors are what initiate _all_ surgeries when you have it installed

Reply:
I'll keep them in, but I'd like to keep the fact that the processor can start surgeries in as QOL

Reply:
yes that's perfectly fine